textattack/loggers/attack_log_manager.py

```diff
@@ -28,11 +30,26 @@ def add_output_file(self, filename):
     def add_output_csv(self, filename, color_method):
         self.loggers.append(CSVLogger(filename=filename, color_method=color_method))
 
+    def hide_failed_results(self):
+        self.hide_failed_result = True
+
+    def hide_skipped_results(self):
+        self.hide_skipped_result = True
+
     def log_result(self, result):
         """ Logs an ``AttackResult`` on each of `self.loggers`. """
         self.results.append(result)
         for logger in self.loggers:
-            logger.log_attack_result(result)
+            if self.hide_failed_result:
+                if not isinstance(result, FailedAttackResult):
+                    logger.log_attack_result(result)
+            if self.hide_skipped_result:
+                if not isinstance(result, SkippedAttackResult):
+                    logger.log_attack_result(result)
+            if self.hide_failed_result and self.hide_skipped_result:
+                if (not isinstance(result, FailedAttackResult)) and \
+                        (not isinstance(result, SkippedAttackResult)):
+                    logger.log_attack_result(result)
```

Comment:
what happens if `(not self.hide_failed_result) and (not self.hide_skipped_result)`? I think the logic here needs to be updated

Reply:
I would suggest first checking if result is `FailedAttackResult` and then checking `self.hide_failed_result` before logging the results.